lms/templates/admin/base.html.jinja2

```diff
@@ -154,6 +154,6 @@
                 {% block content %}{% endblock %}
             </div>
         </section>
+        {% block extra_scripts %}{% endblock %}
```

Comment:
I moved this inside `body`, as HTML does not allow elements outside of `body` or `html`, and browsers are rendering it inside the body anyway.